applications/connectivity_bridge/src/modules/usb_bulk_commands.c

```diff
@@ -125,6 +137,23 @@ size_t dap_execute_cmd(uint8_t *in, uint8_t *out)
 		}
 		goto success;
 	}
+	if (in[0] == ID_DAP_VENDOR_NRF53_VERSION) {
+		const struct fw_info *s0_info = fw_info_find(s0_address_read());
+		const struct fw_info *s1_info = fw_info_find(s1_address_read());
+
+		__ASSERT_NO_MSG(s0_info != NULL);
+		__ASSERT_NO_MSG(s1_info != NULL);
+		int len = snprintf(out + 2, 64 - 2,
+				   "S0: %u, S1: %u, app: " NCS_COMMIT_STRING,
+				   s0_info->version, s1_info->version);
+		if (len < 0) {
```

Comment:
You should also check if `len` exceeds the size of `out`. snprintf can return the size of data that would be been written if the buffer was large enough.

As a side note, it would be nice if the buffer size was given by a function argument, or at the very least a define.